src/main/java/gov/cms/madie/services/PackagingService.java

```diff
@@ -44,9 +44,18 @@ public byte[] createMeasurePackage(Measure measure, String accessToken) {
           measure.getEcqmTitle() + "-v" + measure.getVersion() + "-QDM" + ".html",
           humanReadable.getBytes());
     }
-    String hqmf = hqmfService.generateHqmf((QdmMeasure) measure);
-    entries.put(
-        measure.getEcqmTitle() + "-v" + measure.getVersion() + "-QDM" + ".xml", hqmf.getBytes());
+    // TODO: remove this after HQMF is complete, but for now don't prevent export if HQMF fails
+    try {
+      String hqmf = hqmfService.generateHqmf((QdmMeasure) measure, accessToken);
+      entries.put(
+          measure.getEcqmTitle() + "-v" + measure.getVersion() + "-QDM" + ".xml", hqmf.getBytes());
+    } catch (Exception ex) {
+      // TODO: this is temporary - remove it after!!
+      log.error("An error occurred during HQMF generation!", ex);
```

Comment:
Adding a measure ID in the log statements will help us finding issues on Splunk

Reply:
done